cachito/web/static/api_v1.yaml

```diff
@@ -1049,7 +1049,10 @@ components:
       type: object
       properties:
         environment_variables:
-          $ref: "#/components/schemas/EnvironmentVariable"
+          type: object
+          properties:
+            NAME:
```

Comment:
Out of curiosity, do you know what NAME being uppercase means?

But since the same thing is used as the /requests/id/environment-variables response, I'm going to assume this is correct :smile:

Reply:
Good question. I had initially just replicated what was in the /requests/id/environment-variables response and functionally It appeared to work well and the redoc/swagger output looked fine.

After digging a little deeper though, I think the right approach would be to switch both to use the [additionalProperties](https://swagger.io/docs/specification/data-models/dictionaries/) keyword and drop that NAME key. I'm testing that now.